examples/00-fluent/modeling_cavitation.py

```diff
@@ -0,0 +1,309 @@
+# for the momentum and volume fraction and 'presto!' for pressure.
+
+methods = solver.solution.methods
+
+discretization_scheme = {
+    "k": "first-order-upwind",
+    "mom": "quick",
+    "mp": "quick",
+    "omega": "first-order-upwind",
+    "pressure": "presto!",
+}
+
+methods.discretization_scheme = discretization_scheme
+
+###############################################################################
+# For the pressure velocity coupling scheme choose 'Coupled'. Set the pseudo
+# time method to 'global time step' and Enable 'High Order Term Relaxation'.
+# Then, set the pseudo time explicit relaxation factor for 'Volume Fraction' to
+# 0.3.
+
+methods.p_v_coupling.flow_scheme = "Coupled"
+
+methods.pseudo_time_method.formulation.coupled_solver = "global-time-step"
+
+methods.high_order_term_relaxation.enable = True
+
+solver.solution.controls.pseudo_time_explicit_relaxation_factor.global_dt_pseudo_relax[
+    "mp"
+] = 0.3
+
+###############################################################################
+# To plot the residuals, enable plotting and set the convergence criteria to
+# 1e-05 for x-velocity, y-velocity, k, omega, and vf-vapor. Enable the specified
+# initial pressure then initialize the solution with hybrid initialization.
+
+solver.tui.solve.monitors.residual.plot("yes")
+
+solver.tui.solve.monitors.residual.convergence_criteria(
+    1e-05, 1e-05, 1e-05, 1e-05, 1e-05, 1e-05
+)
+
+solver.solution.initialization.hybrid_init_options.general_settings.initial_pressure = (
+    True
+)
+
+solver.solution.initialization.hybrid_initialize()
+
+###############################################################################
+# Save and Run
+# ~~~~~~~~~~~~
+# Save the case file 'cav.cas.h5'. Then, start the calculation by requesting
+# 500 iterations. Save the final case file and the data.
+
+solver.file.write(file_name="cav", file_type="case")
+
+solver.solution.run_calculation.iterate(iter_count=500)
+
+solver.file.write(file_name="cav", file_type="case-data")
+
+###############################################################################
+# Post Processing
+# ~~~~~~~~~~~~~~~
+# Create a contour plot for static pressure, turbulent kinetic energy and the
+# volume fraction of water vapor. For each plot enable banded coloring and
+# filled.
+
+solver.results.graphics.contour.create("contour_static_pressure")
+
+contour_static_pressure = {
+    "coloring": {"option": "banded", "smooth": False},
+    "field": "pressure",
+    "filled": True,
+}
+
+solver.results.graphics.contour["contour_static_pressure"] = contour_static_pressure
+
+###############################################################################
+# Mirror the display around the symmetry plane to show the full model.
+
+solver.tui.display.set.mirror_zones(["symm_2", "symm_1"])
+
+###############################################################################
+# .. image:: /_static/cavitation_model_012.png
+#   :width: 500pt
+#   :align: center
```

Comment:
We need them as static images only for the purpose of the documentation.  Yes we could add those commands but I don’t think we could generate the images dynamically during the build and use them for the doc.

Reply:
I added the save image lines. They come out in this format: 
![image](https://github.com/ansys/pyfluent/assets/143635850/e2f9d5d1-bd5a-4c12-8fd9-a3adfda9b7ec)

Reply:
@dnwillia I did not mean to generate the images dynamically and use them for the documentation build, my point was to just add the image output commands so users have an idea of how they might be able to obtain these images through PyFluent. Otherwise, and as things currently stand, when running the example they won't see any images at all, despite all the `graphics` and `contours` commands that are effectively not doing anything if these images aren't written to file or visible anywhere through PyFluent.

I guess users could see those contours if they opened a Fluent session with its own GUI and loaded the saved case file then displayed the contours manually.

And you're right, the other examples are also using graphics commands that don't output anything that is visible to the PyFluent user.

Moving this to a separate issue tracker sounds ok to me, going to do that.

@ansaboutin that is correct, that is what I meant by "the image output isn't going to look perfect".